docs/gettinginvolved/plugins/building-driver.rst

```diff
@@ -12,6 +12,7 @@ These drivers include:
 
 * `OpenStack Cinder <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/cinder.py>`_
 * `Amazon EBS <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/ebs.py>`_
+* `Google Compute Engine PD <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/gce.py>`_
```

Comment:
I always try to mimic Amazon EBS but no one seems to buy it.

Updated to see Persistent Disk.